src/main/java/com/project/saw/event/EventEntity.java

```diff
@@ -9,20 +11,21 @@
 
 import java.time.LocalDate;
 import java.util.Set;
-
+import org.springframework.hateoas.*;
 
 @Data
 @Entity
 @Builder
 @AllArgsConstructor
 @NoArgsConstructor
 @Table(name = "events")
```

Comment:
to nic nie robi w tym przypadku. Chyba że hibernate generuje nazwy tabel upper case a chcesz miec lower case. (Chociaż zazwyczaj konwencja jest taka, że nazwy tabel są wielkimi literami)